docsrc/mux.go

```diff
@@ -732,5 +732,14 @@ func SamplesHandler(prefix string) http.Handler {
 		c24,
 	)
 
+	// TODO: c25, Merge L10n docs
```

Comment:
what's this..

Reply:
https://github.com/qor5/docs/pull/43/files#diff-93e68f9afe63c797ad320768c8c79253aabe044e941663c741efbae9097ce897